tests/conftest.py

```diff
@@ -146,6 +146,13 @@ def function_host_register(sm_host):
     sm_host.register()
 
 
+@pytest.fixture(scope='function')
+def function_host_register_for_local_mode(sm_host):
```

Comment:
the local mode doesn't support the `rhsm_' options, so we have to register the local host before testing.